### DISCUSSION TITLE
Ipv6 scan

### DISCUSSION THREAD

**Discussion by sinars89** [MortezaBashsiz/CFScanner#101] [Q&A]
سلام می خواستم بدونم چجوری می شه بجای ipv4 داخل اسکریپت ipv6 سرچ کنیم؟ من تو فایل سابنت وارد کردم ولی ارور داد که  باید از option 6 استفاده کنی

**Comment by raminrez** [comment]
دوستان آیا امکانش نیست بجای v4 از v6 توی کانفیگ‌ها استفاده کنیم؟
@MortezaBashsiz @tempookian

**Comment by tempookian** [comment]
سلام. این امکان به اسکریپت پایتون اضافه شد. توضیحات رو در README ببینید

عملا تغییری در رویه اجرای اسکریپت نمی‌خواد بدید. فایلتون هم می تونه ترکیب ورژن ۴ و ۶ باشه و ترکیب آیپی و سابنت این دو

@raminrez

**Reply by HosseinR8** [reply]
سلام...لطفا آموزش قدم به قدم  اجرای نسخه پایتون روی ویندوز رو بذارید...من هرکاری کردم نتونستم اجرا کنم...ممنون

**Reply by tempookian** [reply]
سلام. آموزش در اولین فرصت آماده خواهد شد. لطفا بگید دقیقا به چه مشکلی خوردید که سعی کنیم توی آموزش‌ها بگنجونیم
